doc/source/scipyoptdoc.py

```diff
@@ -105,7 +105,10 @@ def remove_arg(arg):
                 if opt_name in args:
                     continue
                 if j >= len(impl_args) - len(impl_defaults):
-                    options.append((opt_name, impl_defaults[len(impl_defaults) - (len(impl_args) - j)]))
+                    options.append(
+                        (opt_name,
+                         impl_defaults[len(impl_defaults) - (len(impl_args) - j)])
```

Comment:
Looks like this could get shorter by using a negative index. Would it just be:
```suggestion
                         impl_defaults[len(impl_args) + j])
```
and that might fit on one line. Could also store the index in a variable; that's logical enough.